User.spec.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+const User = require('./User'); 
+
+describe('userModel', () => {
+    beforeAll(() => {
+        return mongoose.connect('mongodb://localhost/testuserdb');
+    })
+
+    afterEach(() => {
+        return User.remove();
+    })
+
+    afterAll(() => {
+        return mongoose.disconnect();
+    })
+
+    //POST
+    describe('create user', () => {
+        it('should create users successfully', async () => {
```

Comment:
Thinking about the db response made me realize that we can have a check for the _id that mongo provides automatically.